src/cairoWriter/writers/cairoContractWriter.ts

```diff
@@ -85,7 +85,7 @@ export class CairoContractWriter extends CairoASTNodeWriter {
         WARP_STORAGE: LegacyMap::<felt252, felt252>,
         WARP_USED_STORAGE: felt252,
         WARP_NAMEGEN: felt252,
-        ${otherStorageVars.map((v) => `${writer.write(v)}`).join('\n')}
+        ${otherStorageVars.map((v) => `${writer.write(v)},`).join('\n')}
```

Comment:
Why? Won't that break when there is more than two otherStorageVars?

Reply:
No, It actually broke the former way with more than 2 vars because it will write something like this
```
    struct Storage {
    WARP_STORAGE: LegacyMap::<felt252, felt252>,
    WARP_USED_STORAGE: felt252,
    WARP_NAMEGEN: felt252,
    a: S
    b: S
```
With one it work just fine because that last comma is kind of optional but with more than 2 there's no separator.
This will be written with the change:
```
    struct Storage {
    WARP_STORAGE: LegacyMap::<felt252, felt252>,
    WARP_USED_STORAGE: felt252,
    WARP_NAMEGEN: felt252,
    a: S,
    b: S,
```

Reply:
My bad, I've read diff the reverse way, though you were deleting the comma